src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -73,22 +77,26 @@ Each issue preference option has its own advantages. You can select any preferen
 
       <td>
         Only one issue will be open at a time for the entire policy.
+      </td>
 
+      <td>
         * Fewest number of notifications
         * Requires immediate action and closing the issues to be effective
       </td>
     </tr>
 
     <tr>
       <td>
-        [One issue per condition](#preference-condition)
+        [One issue per condition and signal](#preference-condition)
```

Comment:
This is incorrect. There is a "one issue per condition" setting that is different than the "one issue per condition and signal" setting.

Reply:
yes, sorry, my bad. Fixed!